### PR TITLE
[availability] Removed unused calendar fetch request

### PR DESCRIPTION
**Pull request by ozsivanov**
# Code changes

- The calendar fetch that was happening in the `onMount` has been removed since it wasn't being used for anything

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: commons/src/types/Availability.ts [hunk 1]
```diff
@@ -29,7 +29,6 @@ export interface Manifest extends NylasManifest {
   event_buffer: number;
   free_color: string;
   mandate_top_of_hour: boolean;
-  max_bookable_slots: number;
```
Comment (philrenaud) [inline commons/src/types/Availability.ts:32]: Is this just the availability manifest max_bookable_slots? Or does this also need to be removed from ScheduleEditor/Booking?

**Review by philrenaud — APPROVED** [review]

File: commons/src/types/Availability.ts (hunk 1 above)
Comment (ozsivanov) [reply commons/src/types/Availability.ts:32]: Good point, I'll remove it everywhere

File: commons/src/types/Availability.ts [hunk 2]
```diff
@@ -105,12 +104,12 @@ export interface BookableSlot extends TimeSlot {
...
-  | "weekdays"
-  | "biweekly"
-  | "weekly"
-  | "monthly";
+    | "none"
+    | "daily"
+    | "weekdays"
+    | "biweekly"
+    | "weekly"
+    | "monthly";
```
Comment (ozsivanov) [inline commons/src/types/Availability.ts:112]: Prettier giveth, prettier taketh away

File: components/availability/src/test.js [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+  );
+
+  const event = {
+    event_title: "My Demo Meeting",
+    event_description: "This is an event created by a demo app!",
+    slot_size: 15,
+    participantEmails: ["demo@nylas.com"],
+  };
+
+  component.participants = [];
```
Comment (heathergh) [inline components/availability/src/test.js:17]: ✂️

**Review by heathergh — APPROVED** [review]
LGTM! My only request is that a ticket be created to update the dashboard

**Pull request merged**